releases/CHANGELOG-v0.68.0.md

```diff
@@ -0,0 +1,41 @@
+
+<a name="v0.68.0"></a>
+# [v0.68.0] - 2022-01-11
+### Bug Fixes 🐞
+- Azure CNI IP address allocation for win nodes ([#4791](https://github.com/Azure/aks-engine/issues/4791))
+- remove api-server insecure flag for 1.20+ ([#4782](https://github.com/Azure/aks-engine/issues/4782))
+- Update Windows dynamic port range ([#4767](https://github.com/Azure/aks-engine/issues/4767))
+
+### Continuous Integration 💜
+- removing flaky log rotation test for Windows nodes ([#4800](https://github.com/Azure/aks-engine/issues/4800))
+- extend sas token expiration for publishing logos ([#4801](https://github.com/Azure/aks-engine/issues/4801))
+
+### Documentation 📘
+- clarify CSI support note on air-gapped ASH ([#4774](https://github.com/Azure/aks-engine/issues/4774))
+
+### Features 🌈
+- only download nvidia drivers if needed ([#4797](https://github.com/Azure/aks-engine/issues/4797))
+- add support for Kubernetes v1.22.5 ([#4789](https://github.com/Azure/aks-engine/issues/4789))
+- add support for Kubernetes v1.20.14 ([#4785](https://github.com/Azure/aks-engine/issues/4785))
+- add support for Kubernetes v1.23.1 ([#4790](https://github.com/Azure/aks-engine/issues/4790))
+- add support for Kubernetes v1.21.8 ([#4786](https://github.com/Azure/aks-engine/issues/4786))
```

Comment:
I am assuming that this has the fix to cloud provider that was messing with load balancer config in the VMSS?  (Removing and adding it over and over again)

Reply:
Nope, there's no released version of that yet. Though they did finally merge to the release branches yesterday!

Those fixes are provided via using the external (out-of-tree) cloudprovider configurations.